js/nativeplot/Index.svelte

```diff
@@ -542,7 +544,15 @@
 					titleFontWeight: "normal",
 					titleFontSize: text_size_sm,
 					labelFontWeight: "normal",
-					offset: 2
+					offset: 2,
+					columns:
+						chart_element.offsetWidth > 0
+							? Math.max(
```

Comment:
what's this for?

Reply:
Sometimes the chart_elment is not innitialized and chart_element.offsetWidth would be 0. Not sure why exactly, but this seems to have fixed it.